web/src/admin/policies/geoip/GeoIPPolicyForm.ts

```diff
@@ -0,0 +1,125 @@
+import { BasePolicyForm } from "@goauthentik/admin/policies/BasePolicyForm";
+import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
+import "@goauthentik/elements/ak-dual-select";
+import { DataProvision } from "@goauthentik/elements/ak-dual-select/types";
+import "@goauthentik/elements/forms/FormGroup";
+import "@goauthentik/elements/forms/HorizontalFormElement";
+import "@goauthentik/elements/forms/SearchSelect";
+
+import { msg } from "@lit/localize";
+import { TemplateResult, html } from "lit";
+import { customElement } from "lit/decorators.js";
+
+import { GeoIPPolicy, Iso3166Api, PoliciesApi } from "@goauthentik/api";
+
+@customElement("ak-policy-geoip-form")
+export class GeoIPPolicyForm extends BasePolicyForm<GeoIPPolicy> {
+    loadInstance(pk: string): Promise<GeoIPPolicy> {
+        return new PoliciesApi(DEFAULT_CONFIG).policiesGeoipRetrieve({
+            policyUuid: pk,
+        });
+    }
+
+    async send(data: GeoIPPolicy): Promise<GeoIPPolicy> {
+        if (data.asns?.toString() === "") {
+            data.asns = [];
+        } else {
+            data.asns = (data.asns as unknown as string).split(",").map(Number);
+        }
```

Comment:
The actual type of `data` slightly differs from `GeoIPPolicy`.

I haven't found an easy way to reconcile them (the "proper" way is probably building a component on top of `<input>` to run a transforming function when its `value` is read), so I opted for the type assertion.